back/app/models/phase.rb

```diff
@@ -112,6 +114,17 @@ def permission_scope
     self
   end
 
+  def last_phase?
+    phases = Phase.where(project_id: project_id)
```

Comment:
Tried using project.phases, but about 80 tests failed due to them not adding phases to the project with `project.phases << phase`